tests/tests_test_workflow/test_benchmark_workflow/benchmark_test/test_benchmark_test_suite.py

```diff
@@ -35,20 +36,21 @@ def test_execute_security_enabled(self) -> None:
         with patch("subprocess.check_call") as mock_check_call:
             benchmark_test_suite.execute()
             self.assertEqual(mock_check_call.call_count, 1)
-            self.assertEqual(benchmark_test_suite.command, 'docker run opensearchproject/opensearch-benchmark:latest execute_test '
-                                                           '--workload=nyc_taxis --test-mode --pipeline=benchmark-only '
-                                                           '--target-hosts=abc.com --client-options="use_ssl:true,'
-                                                           'verify_certs:false,basic_auth_user:\'admin\',basic_auth_password:\'admin\'"')
+            self.assertEqual(benchmark_test_suite.command,
+                             'docker run opensearchproject/opensearch-benchmark:latest execute_test '
+                             '--workload=nyc_taxis --test-mode --pipeline=benchmark-only '
+                             '--target-hosts=abc.com --client-options="use_ssl:true,'
+                             'verify_certs:false,basic_auth_user:\'admin\',basic_auth_password:\'admin\'"')
 
     def test_execute_default_with_optional_args(self) -> None:
-        TestBenchmarkTestSuite.setUp(self, "/home/test/benchmark.ini", "key1:value1,key2:value2")
+        TestBenchmarkTestSuite.setUp(self, "/home/test/benchmark.ini", "key1:value1,key2:value2", "number_of_replicas:1")
         with patch("subprocess.check_call") as mock_check_call:
             self.benchmark_test_suite.execute()
             self.assertEqual(mock_check_call.call_count, 1)
-
             self.assertEqual(self.benchmark_test_suite.command, 'docker run -v /home/test/benchmark.ini:'
                                                                 '/opensearch-benchmark/.benchmark/benchmark.ini '
                                                                 'opensearchproject/opensearch-benchmark:latest execute_test '
                                                                 '--workload=nyc_taxis --test-mode '
                                                                 '--pipeline=benchmark-only --target-hosts=abc.com '
+                                                                '--workload-params "number_of_replicas:1" '
```

Comment:
If there are more than 1 workload params how should that be passed?

Reply:
`--workload-params "number_of_replicas:1,number_of_shards:5"`

Reply:
Should we add that in description or example? here https://github.com/opensearch-project/opensearch-build/pull/3481/files#diff-387b43a35cba3276196198afe8c1473e4d72d9b62438a2611cbe192f827396c3R79-R80